src/main/kotlin/com/github/oxc/project/oxcintellijplugin/settings/OxcConfigurable.kt

```diff
@@ -0,0 +1,185 @@
+
+                    val detectAutomaticallyHelpText =
+                        JavaScriptBundle.message("settings.javascript.linters.autodetect.configure.automatically.help.text",
+                            ApplicationNamesInfo.getInstance().fullProductName,
+                            displayName,
+                            "${OxcPackage.CONFIG_NAME}.json")
+
+                    val helpLabel = ContextHelpLabel.create(detectAutomaticallyHelpText)
+                    helpLabel.border = JBUI.Borders.emptyLeft(UIUtil.DEFAULT_HGAP)
+                    cell(helpLabel)
+                }
+                row {
+                    manualConfiguration =
+                        radioButton(JavaScriptBundle.message("settings.javascript.linters.autodetect.configure.manually",
+                            displayName)).bindSelected(ConfigurationModeProperty(settings,
+                            ConfigurationMode.MANUAL)).component
+                }
+            }
+
+            // *********************
+            // Manual configuration row
+            // *********************
+            panel {
+                row(OxcBundle.message("oxc.settings.languageServerPath")) {
+                    textFieldWithBrowseButton(OxcBundle.message("oxc.settings.languageServerPath")) {
+                        it.path
+                    }.bindText(settings::binaryPath)
+                }.visibleIf(manualConfiguration.selected)
+
+                row(OxcBundle.message("oxc.config.path.label")) {
+                    textFieldWithBrowseButton(
+                        OxcBundle.message("oxc.config.path.label"),
+                        project,
+                    ) { it.path }.bindText(settings::configPath)
+                }.visibleIf(manualConfiguration.selected)
+            }
+
+//            row(MyBundle.message("oxc.settings.oxlintRunTrigger")) {
+//                comboBox(listOf(OxlintRunTrigger.ON_SAVE, OxlintRunTrigger.ON_TYPE)).bindItem({
+//                    return@bindItem settings.runTrigger
+//                }, {
+//                    if (it != null) {
+//                        settings.runTrigger = it
+//                    }
+//                })
+//            }.enabledIf(!disabledConfiguration.selected)
+
+            // *********************
+            // Supported file extensions row
+            // *********************
+            row(OxcBundle.message("oxc.supported.extensions.label")) {
+                extensionsField = textField().align(AlignX.FILL)
+                    .bindText({ settings.supportedExtensions.joinToString(",") }, { value ->
+                        settings.supportedExtensions =
+                            value.split(",").map { it.trim() }.filter { it.isNotBlank() }.toMutableList()
+                    }).validationOnInput { validateExtensions(it) }.applyToComponent {
+                        font = font.deriveFont(font.size2D - 2f) // Reduce font size by 2 points
```

Comment:
I feel like the font is kind of small, but it can probably be left alone for now.

![image](https://github.com/user-attachments/assets/92424849-7b09-4f12-a8bc-2dd541ec3f31)

Reply:
I agree, but I don’t have a better idea. We can try another font or control if it doesn’t work for us